chpl/chpl-service/src/test/java/gov/healthit/chpl/permissions/domain/activity/GetUserActivityActionPermissionsTest.java

```diff
@@ -0,0 +1,98 @@
+package gov.healthit.chpl.permissions.domain.activity;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import gov.healthit.chpl.permissions.ResourcePermissions;
+import gov.healthit.chpl.permissions.domain.ActionPermissionsBaseTest;
+import gov.healthit.chpl.permissions.domains.activity.GetByAcbActionPermissions;
+
+public class GetUserActivityActionPermissionsTest extends ActionPermissionsBaseTest {
+
+    @Mock
+    private ResourcePermissions resourcePermissions;
+
+    @InjectMocks
+    private GetByAcbActionPermissions permissions;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+
+        Mockito.when(resourcePermissions.getAllAcbsForCurrentUser()).thenReturn(getAllAcbForUser(2l, 4l));
+    }
+
+    @Override
+    @Test
+    public void hasAccess_Admin() throws Exception {
+        setupForAdminUser(resourcePermissions);
+
+        assertTrue(permissions.hasAccess());
+
+        // Not used
+        assertFalse(permissions.hasAccess(new Object()));
+    }
+
+    @Override
+    @Test
+    public void hasAccess_Onc() throws Exception {
+        setupForOncUser(resourcePermissions);
+
+        assertTrue(permissions.hasAccess());
+
+        // Not used
+        assertFalse(permissions.hasAccess(new Object()));
+    }
+
+    @Override
+    @Test
+    public void hasAccess_Acb() throws Exception {
+        setupForAcbUser(resourcePermissions);
+
+        assertTrue(permissions.hasAccess());
```

Comment:
I didn't think ROLE_ACB had user activity access. I know they don't in the UI; are they allowed in the API?